web/middleware.ts

```
@@ -0,0 +1,106 @@
+import { createMiddlewareClient } from "@supabase/auth-helpers-nextjs"
+import { NextResponse } from "next/server"
+import type { NextRequest } from "next/server"
+
+export async function middleware(req: NextRequest) {
+  const res = NextResponse.next()
+
+  try {
+    // Check if we're using placeholder credentials from environment variables
+    const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL || ""
+    const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY || ""
+    const isUsingRealCredentials = supabaseUrl && supabaseUrl.startsWith("http") && supabaseAnonKey
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)